# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/mp4_demuxer/TestMP4.cpp
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 dom/media/gtest/mp4_demuxer/TestMP4.cpp
--- a/dom/media/gtest/mp4_demuxer/TestMP4.cpp	Fri Jan 27 19:06:16 2023 +0000
+++ b/dom/media/gtest/mp4_demuxer/TestMP4.cpp	Fri Jan 27 21:11:42 2023 +0200
@@ -92,12 +92,12 @@
   io = {vector_reader, &buf};
   rv = mp4parse_new(&io, &parser);
   ASSERT_EQ(parser, nullptr);
-  EXPECT_EQ(rv, MP4PARSE_STATUS_MOOV_MISSING);
+  EXPECT_EQ(rv, MP4PARSE_STATUS_NO_MOOV);
 
   buf.buffer.reserve(4097);
   rv = mp4parse_new(&io, &parser);
   ASSERT_EQ(parser, nullptr);
-  EXPECT_EQ(rv, MP4PARSE_STATUS_MOOV_MISSING);
+  EXPECT_EQ(rv, MP4PARSE_STATUS_NO_MOOV);
 
   // Empty buffers should fail.
   buf.buffer.resize(4097, 0);